README.md

```diff
@@ -141,6 +142,25 @@ Sentry as part of the extra data, if it was the last action before an error.
 
 This option was introduced in version 1.1.1.
 
+#### `getUserContext` _(Function)_
+
+Default: `undefined` (disabled)
+
+Signature: `state => userContext`
```

Comment:
Not sure if this is the best way to do this, and still be consistent with the others. Since it cannot default to `state => undefined`; because it should be possible not to use it, and use the `Raven.setUserContext` elsewhere....

Reply:
How about we specify the signature as:

    #### `getUserContext` _(Optional Function)_

And then omit the “default” line.

Reply:
Sounds good :+1: Fixed it now :smile: